Modules/Scripted/SampleData/SampleData.py

```diff
@@ -483,6 +490,16 @@ def registerDevelopmentSampleDataSources(self):
       checksums=['SHA256:c0743772587e2dd4c97d4e894f5486f7a9a202049c8575e032114c0a5c935c3b', 'SHA256:3243b62bde36b1db1cdbfe204785bd4bc1fbb772558d5f8cac964cda8385d470']
       )
 
+  def registerTestingDataSources(self):
+    """Fills in the sample data sources displayed only if developer mode is enabled."""
```

Comment:
``Fills in the sample data sources displayed only if developer mode is enabled.` -> `Register sample data sources used by SampleData self-test to test module functionalities.`